json/demographics.json

```diff
@@ -87,7 +87,7 @@
                 "title": "demographics_people_leaving_community",
                 "uiStateJSON": "{\"vis\":{\"params\":{\"sort\":{\"columnIndex\":null,\"direction\":null},\"config\":{\"searchKeyword\":\"\"}}}}",
                 "version": 1,
-                "visState": "{\"title\":\"demographics_people_leaving_community\",\"type\":\"table\",\"params\":{\"perPage\":5,\"showPartialRows\":false,\"showMeticsAtAllLevels\":false,\"sort\":{\"columnIndex\":null,\"direction\":null},\"showTotal\":false,\"totalFunc\":\"sum\"},\"aggs\":[{\"id\":\"3\",\"enabled\":true,\"type\":\"min\",\"schema\":\"metric\",\"params\":{\"field\":\"demography_max_date\",\"customLabel\":\"Last Contrib Date\"}},{\"id\":\"2\",\"enabled\":true,\"type\":\"date_range\",\"schema\":\"bucket\",\"params\":{\"field\":\"demography_max_date\",\"ranges\":[{\"from\":\"now-12M\",\"to\":\"now-6M\"}],\"customLabel\":\"Period of getting inactive (last contrib 6 months ago)\"}},{\"id\":\"1\",\"enabled\":true,\"type\":\"count\",\"schema\":\"metric\",\"params\":{\"customLabel\":\"Contribs\"}},{\"id\":\"4\",\"enabled\":true,\"type\":\"terms\",\"schema\":\"bucket\",\"params\":{\"field\":\"author_name\",\"size\":30,\"order\":\"desc\",\"orderBy\":\"3\",\"customLabel\":\"Author\"}}]}"
+                "visState": "{\"title\":\"demographics_people_leaving_community\",\"type\":\"table\",\"params\":{\"perPage\":5,\"showPartialRows\":false,\"showMeticsAtAllLevels\":false,\"sort\":{\"columnIndex\":null,\"direction\":null},\"showTotal\":false,\"totalFunc\":\"sum\"},\"aggs\":[{\"id\":\"3\",\"enabled\":true,\"type\":\"min\",\"schema\":\"metric\",\"params\":{\"field\":\"demography_max_date\",\"customLabel\":\"Last Contrib Date\"}},{\"id\":\"2\",\"enabled\":true,\"type\":\"date_range\",\"schema\":\"bucket\",\"params\":{\"field\":\"demography_max_date\",\"ranges\":[{\"from\":\"now-12M\",\"to\":\"now-6M\"}],\"customLabel\":\"Period of becoming inactive (last contrib 6 months ago)\"}},{\"id\":\"1\",\"enabled\":true,\"type\":\"count\",\"schema\":\"metric\",\"params\":{\"customLabel\":\"Contribs\"}},{\"id\":\"4\",\"enabled\":true,\"type\":\"terms\",\"schema\":\"bucket\",\"params\":{\"field\":\"author_name\",\"size\":30,\"order\":\"desc\",\"orderBy\":\"3\",\"customLabel\":\"Author\"}}]}"
```

Comment:
I know this is not part of the changes included in this PR but `"Period of becoming inactive (last contrib 6 months ago)\"` sounds odd to me. Shouldn't it be `Threshold to be considered inactive (last contrib 6 months ago)`

Reply:
I believe this is a table heading, which shows above the column depicting the time period of inactivity with start date - end date.

Therefore, perhaps it should say 'Inactive between these dates' with the sub text in the panel at the bottom explaining that this is calculated based on 6 months of inactivity.

Even better if there was some way to select the time frame you want to be applied (as half a year is a long time to wait until you notice a contributor dropping off!)

As a side note, on my dash I have somewhat conflicting data, for example below this shows the last commit on 27th March 2019 within the time period stated on the first column? Maybe I'm not clear what this data is showing?

Oct 10th 2018, 18:25 to Apr 10th 2019, 18:25 | Name | 39 | Mar 27th 2019, 18:04

Reply:
That makes sense, thanks for the clarification! The rationale also makes sense, although my question was, perhaps 6 months is a little too long! But maybe not!

Reply:
> That makes sense, thanks for the clarification! The rationale also makes sense, although my question was, perhaps 6 months is a little too long! But maybe not!

It might be. As it is something fixed, I think opening an issue proposing to change this by a different period could make sense. 

Exploring whether a dashboard for people leaving, based on `demography_max_date`, could make sense is another option. When you select a time range on the top right corner all data in the dashboard get filtered by that range. Demographics dashboard is based on `creation_date`. For that reason is set to 10y, making sure we include the whole history of the community.